broker/cloud_run/lsst/lite/main.py

```diff
@@ -0,0 +1,170 @@
+    )
+    source_lite_dict = _create_lite_dict(
+        alert.dict.get(alert.get_key("source")), _create_source_fields_list(alert)
+    )
+    alert_lite_dict = {
+        alert.get_key("alertid"): alert.alertid,
+        alert.get_key("source") + "-lite": source_lite_dict,
+        alert.get_key("prv_sources") + "-lite": prev_sources_lite_dict,
+        alert.get_key("object") + "-lite": object_lite_dict,
+    }
+
+    return pittgoogle.Alert.from_dict(payload=alert_lite_dict, schema_name=f"{SURVEY}.lite")
+
+
+def _create_source_fields_list(alert: pittgoogle.Alert) -> list[str]:
+    """Creates a list of survey-specific field names to be included in the lite alert for the source dictionary."""
+    broker_field_names = [
+        "sourceid",
+        "mjd",
+        "ra",
+        "ra_err",
+        "dec",
+        "dec_err",
+        "flux",
+        "flux_err",
+        "filter",
+    ]
+
+    return _get_survey_field_names(alert, broker_field_names)
+
+
+def _create_object_fields_list(alert: pittgoogle.Alert) -> list[str]:
+    """Creates a list of survey-specific field names to be included in the lite alert for the object dictionary."""
+    broker_field_names = [
+        "objectid",
+        "nearby_object1",
+        "nearby_object2",
+        "nearby_object3",
+        "nearby_object1_distance",
+        "nearby_object2_distance",
+        "nearby_object3_distance",
+        "prob_is_object1",
+        "prob_is_object2",
+        "prob_is_object3",
+        "u_flux_err_mean",
+        "g_flux_err_mean",
+        "r_flux_err_mean",
+        "i_flux_err_mean",
+        "z_flux_err_mean",
+        "y_flux_err_mean",
+    ]
+
+    return _get_survey_field_names(alert, broker_field_names)
+
+
+def _get_survey_field_names(alert: pittgoogle.Alert, broker_field_names: list) -> list[str]:
+    """Returns a list of survey-specific field names to be included in the lite dictionary."""
+    _survey_field_names = [alert.get_key(field) for field in broker_field_names]
+
+    # fields may be lists, extract the second element
+    survey_field_names = [
+        _survey_field_name[1] if isinstance(_survey_field_name, list) else _survey_field_name
+        for _survey_field_name in _survey_field_names
+    ]
+
+    return survey_field_names
+
+
+def _create_lite_dict(alert_dict: dict, field_names: list[str]) -> dict:
+    return _drop_fields(alert_dict, field_names)
+
+
+def _create_prv_sources_dict(
+    source_history: list[dict], field_names: list[str]
+) -> Optional[list[dict]]:
+    """Create a list of prv_sources dictionaries if they exist."""
+
+    if source_history is None:
+        return source_history
+
+    prev_sources = []
+    for prv_s in source_history:
+        lite_source_dict = _drop_fields(prv_s, field_names)
+        prev_sources.append(lite_source_dict)
+
+    return prev_sources
+
+
+def _drop_fields(alert_dict: dict, field_names: list[str]) -> dict:
```

Comment:
I suggest dropping this function and just using line 168 in its place. If you do really want this to be a function, then I recommend changing the name because the signature makes me think that whatever is passed to `field_names` is going to be dropped.